plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2336,8 +2336,21 @@ elementgui.block.properties_redstone=Redstone properties
 elementgui.block.redstone_connect=<html>Does redstone connect to this block?<br><small>Power works even if this is unchecked
 elementgui.block.emits_redstone=Does emit redstone?
 elementgui.block.redstone_power=Emitted redstone power:
+elementgui.block.custom_properties_states=Custom block properties
+elementgui.block.custom_properties.add=Add custom property
+elementgui.block.custom_properties.add.title=New block state property
+elementgui.block.custom_properties.add.input=Property name
+elementgui.block.custom_properties.add.error_duplicate=Property with this name is already defined.
+elementgui.block.custom_properties.add.error_duplicate.title=Duplicate property
+elementgui.block.custom_properties.error_overrides_block_base=Some defined custom properties are already used by the selected block base.
```

Comment:
Adapt this message so it will include properties like axis or waterlogged provided by parameters of the block base